python/task/download.py

```diff
@@ -51,8 +64,9 @@ def run(self, task):
         f.write(chunk)
         task.increment(total_bytes, len(chunk))
         task.increment(bytes, len(chunk))
+    if ratelimit > 0: download_concurrency -= 1 
     log.info(name + " downloaded")
-
+	
```

Comment:
Please set your editor to remove trailing white space.

Reply:
done